python/ray/tune/tune.py

```diff
@@ -98,12 +98,11 @@ def run(run_or_experiment,
         trial_executor=None,
         raise_on_failed_trial=True,
         return_trials=False,
-        ray_auto_init=True,
-        sync_function=None):
+        ray_auto_init=True):
     """Executes training.
 
     Args:
-        run_or_experiment (function|class|str|Experiment): If
+        run_or_experiment (function | class | str | :class:`Experiment`): If
```

Comment:
Should we use `Union[a,b,c]` here, instead?

Reply:
Not sure... I'm still a little unfamiliar with Unions myself. Will keep in mind.